tensorflow/stream_executor/rocm/BUILD

```diff
@@ -5,9 +5,11 @@ load(
     "//tensorflow/stream_executor:build_defs.bzl",
     "stream_executor_friends",
 )
+
 load("//tensorflow:tensorflow.bzl", "tf_copts")
-load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm_is_configured")
+load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm_is_configured", "rocm_default_copts")
 load("//tensorflow/core/platform:build_config_root.bzl", "if_static")
+load("@local_config_cuda//cuda:build_defs.bzl","cuda_library")
```

Comment:
I'm not sure about this. Looks a bit hacky to import Cuda tool chain. Does this means we need to extend our toolchain? @whchung